test/test-helpers/fake-data.ts

```diff
@@ -147,6 +166,22 @@ export const fakeHost = async (hostData: Parameters<typeof fakeCollective>[0] =
   });
 };
 
+/** Create a fake host */
+export const fakeActiveHost = async (hostData: Parameters<typeof fakeCollective>[0] = {}) => {
```

Comment:
I wanted to make this the default behavior for `fakeHost` since most hosts are configured as active nowadays, but it breaks a few tests so I've introduced it as a separate helper.